### PR TITLE
modifying instructions on how to join mailing list

### PR DESCRIPTION
**Pull request by jibarnum**
The old method for some reason stopped working as it used to... instead of getting to the request to join page, you ended up at a page that showed all the pyhc list emails, but no option to join anywhere. Found a different way that people can join the list, which is also much simpler than the old method. Silver lining.

### CODE REVIEW

**Review by sapols — CHANGES_REQUESTED** [review]

**Comment by sapols** [comment]
@jibarnum I wrote a small comment about capitalizing "PyHC" a few days ago but I'm not sure if you were able to see it before I submitted my review just now. Sorry if you haven't seen anything from me until now.

File: _pages/contact.md [hunk 1]
```diff
@@ -6,16 +6,13 @@ order: 6
 ---
 ## Mailing List
 There is a Google Groups mailing list for discussions open to anyone. To be added to the mailing list for this project, please do the following:<br />
-1) Use the following link to add yourself to the mailing list [this link](https://groups.google.com/group/pyhc-list/subscribe).* <br />
-2) You should receive a follow up email to confirm your email.\** In that email, click 'Join This Group' to confirm your subscription. It should then send you back to the Google Groups page, with a message saying your email has been added to the groups.<br />
+1) Send an email to pyhc-list+subscribe@googlegroups.com. It doesn't have to have any subject/body text.  
+2) Shortly thereafter (generally, less than a minute), you'll get a reply from the pyhc subscribe list. Just hit reply and send. Again, no subject/body text is required.
```
[the remark this answers is not in the document]
Comment (jibarnum) [reply _pages/contact.md:10]: These instructions are so much more succinct, I'm a big fan. 👍 I'll make that word fix, good thought!

**Comment by jibarnum** [comment]
No problem @sapols . Didn't see the comment before now, but no one has seemed to need to subscribe in the last few days, so not a problem.

**Comment by jibarnum** [comment]
Addressed the capitalization fix, going to go ahead and merge.

**Pull request merged**